### PR TITLE
Docs reorganization

### PR DESCRIPTION
**Pull request by philburrrt**
Restructured docs layouts. New copy only where necessary and some obviously needed updates were fixed.

visual here: 
https://philbertburrow.gitbook.io/yearn-docs/v/docs-reorganization/

### CODE REVIEW

**Review by lehnberg — APPROVED** [review]
Awesome work, this looks good to me. Will ask for a couple of more 👀 on it before we merge.

File: README.md [hunk 1]
```diff
@@ -30,7 +22,7 @@ The Yearn ecosystem is controlled by YFI token holders who submit and vote on of
 
 Governance Forum [https://gov.yearn.finance/](https://gov.yearn.finance/)
 
-Discord [http://discord.yearn.finance](http://discord.yearn.finance)
+Discord [http://discord.yearn.finance](https://discord.gg/BUAFD9DUe)
```
Comment (milkyklim) [inline README.md:25]: Keep this link as http://discord.yearn.finance it will be resolved to proper invite on domain manager.

File: SUMMARY.md [hunk 2]
```diff
@@ -1,39 +1,54 @@
...
-- [ySwap](r-and-d/yswap.md)
-- [yTrade](r-and-d/ytrade.md)
-- [Delegated Funding DAO Vaults](r-and-d/delegated-funding-dao-vaults.md)
-- [StableCredit](r-and-d/stablecredit.md)
+- [YFI](governance/yfi.md)
+- [Governance Process](governance/governance-process/README.md)
+  - [Discussion](governance/governance-process/discussion.md)
+  - [Proposal](governance/governance-process/proposal.md)
+  - [Voting](governance/governance-process/voting.md)
+  - [implementation](governance/governance-process/implementation.md)
```
Comment (milkyklim) [inline SUMMARY.md:22]: **I**mplementation

File: governance/governance-process/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Governance Process
+
+The Yearn ecosystem is controlled by YFI token holders who submit and vote on off-chain proposals that govern the ecosystem. Proposals that generate a majority support (>50% of the vote) are implemented by a 9 member multi-signature wallet. Changes must be signed by 6 out of the 9 wallet signers in order to be implemented. The members of the multi-signature wallet were voted in by YFI holders and are subject to change from future governance votes. Please refer to our FAQ for the list of the multisig signers and more information about the governance process.
```
Comment (milkyklim) [inline governance/governance-process/README.md:3]: > Please refer to our FAQ

Can we add a link to FAQ here?

File: governance/governance-process/discussion.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#Discussion 
+
+Discussion regarding changes in the protocol happen on a variety of platforms such as:
+
+ - [Governance Forum](https://gov.yearn.finance/)
+ - [Discord](https://discord.gg/BUAFD9DU)
```
Comment (milkyklim) [inline governance/governance-process/discussion.md:6]: http://discord.yearn.finance

File: governance/governance-process/implementation.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#YIP Implementation
+
+Once a Snapshot vote has received over 50% support after 5 days of voting, changes will be implemented by Yearn's protocol or operations team and signed by the multi-sig, if necessary.
```
Comment (milkyklim) [inline governance/governance-process/implementation.md:3]: Better replace with *majority* cause it's possible people push multi option vote.

File: governance/governance-process/voting.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Voting
+
+## How do I vote?
+
+- Holding or staking YFI in the following places will enable you to vote on Yearn's [Snapshot](https://snapshot.page/#/yearn) page:
```
Comment (milkyklim) [inline governance/governance-process/voting.md:5]: + Bancor

File: governance/governance-process/voting.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+	- Your wallet
```
Comment (milkyklim) [inline governance/governance-process/voting.md:6]: This is not correct, you waller doesn't work

File: governance/governance-process/voting.md (hunk 6 above)
Comment (milkyklim) [reply governance/governance-process/voting.md:5]: Here is the exact list: https://github.com/banteg/guest-list/pull/10#issue-599647475

File: README.md (hunk 1 above)
Comment (listonjesse) [reply README.md:25]: Yes, I changed this with my earlier PR, but now the link is fixed by @nymmrx so let's revert back to it

File: README.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# Introduction to Yearn
+# Introduction
 
 Yearn Finance is a suite of products in Decentralized Finance (DeFi) that provides lending aggregation, yield generation, and insurance on the Ethereum blockchain. The protocol is maintained by various independent developers and is governed by YFI holders.
```
Comment (DarkGhost7) [inline README.md:3]:
```suggestion
Yearn Finance is a suite of products in Decentralized Finance (DeFi) that provides lending aggregation and yield generation on the Ethereum blockchain. The protocol is maintained by various independent developers and is governed by YFI holders.
```
Do we still provide insurance?

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]

File: resources/links.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+* [Github](https://github.com/yearn)
+* [Governance Forum](https://gov.yearn.finance)
+
+### Learn
+
+* [Learn Yearn](https://www.learnyearn.finance/)
+* [yCosystem](https://ycosystem.info/)
+
+### Vaults Detail Reference
```
Comment (DarkGhost7) [inline resources/links.md:19]:
```suggestion
### Vault Strategies

* [Vaults at Yearn](https://medium.com/yearn-state-of-the-vaults/the-vaults-at-yearn-9237905ffed3)
* [yVaults v2 Strategies](https://yearn-hub.vercel.app/)

### Statistics

* [Yearn Dashboard](https://yearn.vision)
* [Yearn TVL](https://yearn.science/)
* [Initial Distribution Dashboard](https://explore.duneanalytics.com/dashboard/yearn)
* [Voting Stats Dashboard](https://explore.duneanalytics.com/public/dashboards/Lqnxzm7fa8NVhKC4kc37DDFPZgqXryaIjyLRYAYp)
* [Vaults Stats Dashboard](https://explore.duneanalytics.com/public/dashboards/g0bGfgloeXBd9C18jpBjdXi5KkQjR7IXYqFRUnHk)
* [Yearn API](https://vaults.finance/all)

```

Comment (DarkGhost7) [reply resources/links.md:19]: Deleted outdated links and added newer ones that are maintained actively.

**Review by DarkGhost7 — CHANGES_REQUESTED** [review]

File: resources/links.md (hunk 9 above)
Comment (DarkGhost7) [reply resources/links.md:19]: if it's not clear all this should be deleted
### Vaults Detail Reference

* [https://vaults.finance/](https://vaults.finance/)

* [yieldfarming.info](https://yieldfarming.info/)
* [stats.finance/yearn](https://stats.finance/yearn)
* [Feel The Yearn](https://feel-the-yearn.vercel.app/)

Comment (philburrrt) [reply resources/links.md:19]: Thanks, fixed it.

**Review by milkyklim — APPROVED** [review]

**Pull request closed without merging**